test/index.ts

```diff
@@ -43,12 +43,23 @@ for (let envVar of Object.keys(environmentVariables)) {
     if (match) {
         let [, option] = match;
         let value: string | number = environmentVariables[envVar];
-        if (typeof value === 'string' && !isNaN(parseInt(value, undefined))) {
-            value = parseInt(value, undefined);
+
+        try {
+            value = JSON.parse(value);
+        } catch
```

Comment:
It took me a bit of time to realize this catch block does not intend to use anything from the thrown error. Is this syntax common?

Reply:
It's a little unusual having the extra code in the catch. There's no tryParse so I just have to try it.

It's going away, though.